C/plugins/storage/postgres/connection.cpp

```diff
@@ -1581,7 +1583,9 @@ bool 		add_row = false;
 
 			// Handles - asset_code
 			sql.append(",\'");
-			sql.append((*itr)["asset_code"].GetString());
+			std::string escapedAsset((*itr)["asset_code"].GetString());
+			escapedAsset = std::regex_replace(escapedAsset, std::regex("\'"), "\'\'");
```

Comment:
Regex is a really slow way of doing this. We used to make more use of regex but removed it because of the big impact it had on performance. You are not using any regular expressions here, so there are much simpler and more performant ways to do this.